docs/stories/EPIC-Restore deleted messages/TF-1719-User-can-restore-deleted-messages.md

```diff
@@ -25,11 +25,11 @@ https://github.com/linagora/tmail-flutter/issues/1719
       + When I select Custom range, a Date picker will be displayed that allow me to select From date and To date.
       + The date range is not limited 
    - Subject: a text field 
-   - Recipient: I can input name/email address, the system will display a suggestion list of matched emails. I can select from the list or input the recipient mannually 
-   - Sender:  I can input name/email address, the system will display a suggestion list of matched emails. I can select from the list or input the recipient mannually
+   - Recipients: I can input name/email address, the system will display a suggestion list of matched emails. I can select from the list or input the recipients mannually 
+   - Senders:  I can input name/email address, the system will display a suggestion list of matched emails. I can select from the list or input the senders mannually
    - Has attachment: a Checkbox. If it is checked, the system will find and restore email which contains attachments
 - After input conditions, I click button Restore, the system will search for matched messages in in deleted vault. 
-- When the search is in progress, there will be a loading status will message :"Recovery is in progress and might take time. Please wait a few minutes." And a Cancel button. If I click Cancel, the restoring process stops immediately
+- When the search is in progress, there will be a toast message :"Recovery is in progress and might take time. Please wait a few minutes." 
```

Comment:
Edit: I added the "cancelation" feature part of the backen team tasks. So we can keep it.

With @hoangdat we decided that this could take the form of a progress bar.